functions/createField.js

```diff
@@ -36,4 +41,41 @@ export function createField() {
         params.idCollection.add(cell.id);
         coordinateX++;
     }
+
+    if (params.difficultyLevel === 'levelMedium') {
+
+        // верхняя граница игрового поля
+        drawSquareLine(
+            { x: FIELD_CELL_CENTER_X - (FIELD_CELL_CENTER_X - 1), y: NUMBER_SQUARES_ROUNDED_Y },

```

Comment:
Почему такое сложное получение верхнего левого квадрата? Это ведь x=1 сразу ?

```suggestion
            { x: 1, y: NUMBER_SQUARES_ROUNDED_Y },

```